hedera-node/configuration/mainnet/upgrade/feeSchedules.json

```diff
@@ -4673,38 +4761,38 @@
             {
               "subType": "TOKEN_NON_FUNGIBLE_UNIQUE",
               "nodedata": {
-                "constant": 3606440483,
-                "bpt": 5765618,
-                "vpt": 14414043986,
-                "rbh": 3844,
-                "sbh": 288,
-                "gas": 38437,
-                "bpr": 5765618,
-                "sbpr": 144140,
+                "constant": 0,
+                "bpt": 40000000000,
+                "vpt": 376062755,
+                "rbh": 0,
+                "sbh": 0,
+                "gas": 0,
+                "bpr": 0,
+                "sbpr": 0,
                 "min": 0,
                 "max": 1000000000000000
               },
               "networkdata": {
-                "constant": 72128809656,
-                "bpt": 115312352,
-                "vpt": 288280879722,
-                "rbh": 76875,
-                "sbh": 5766,
-                "gas": 768749,
-                "bpr": 115312352,
-                "sbpr": 2882809,
+                "constant": 0,
+                "bpt": 160000000000,
+                "vpt": 6017004081,
+                "rbh": 0,
+                "sbh": 0,
+                "gas": 0,
+                "bpr": 0,
+                "sbpr": 0,
                 "min": 0,
                 "max": 1000000000000000
               },
               "servicedata": {
-                "constant": 72128809656,
-                "bpt": 115312352,
-                "vpt": 288280879722,
-                "rbh": 76875,
```

Comment:
Are these hedera-node changes supposed to be in the state proofs pr?

Reply:
Nope. Merging develop fixed this.